charts/spire/charts/spire-agent/templates/configmap.yaml

```diff
@@ -24,26 +42,28 @@ agent:
 
 plugins:
   NodeAttestor:
-    - k8s_psat:
-        plugin_data:
-          cluster: {{ include "spire-lib.cluster-name" . | quote }}
+    k8s_psat:
+      plugin_data:
+        cluster: {{ include "spire-lib.cluster-name" . | quote }}
 
   KeyManager:
-    - memory:
-        plugin_data:
+    memory:
+      plugin_data:
 
   WorkloadAttestor:
-    - k8s:
-        plugin_data:
-          # Defaults to the secure kubelet port by default.
-          # Minikube does not have a cert in the cluster CA bundle that
-          # can authenticate the kubelet cert, so skip validation.
-          skip_kubelet_verification: {{ .Values.workloadAttestors.k8s.skipKubeletVerification }}
-          disable_container_selectors: {{ .Values.workloadAttestors.k8s.disableContainerSelectors }}
+  {{- if .Values.workloadAttestors.k8s.enabled }}
```

Comment:
Is there ever a situation where you don't want to do k8s attestors in a k8s deployment?

Reply:
I'd like to use it for authenticating a nested spire server with the upstream-agent using just a tpm for extra security. So, I'd turn off the k8s one as it only supports the k8s node attestor